app/src/main/java/com/khmelenko/lab/travisclient/mvp/MvpActivity.java

```diff
@@ -18,6 +19,10 @@
  */
 public abstract class MvpActivity<T extends MvpPresenter> extends AppCompatActivity {
 
+    static {
+        AppCompatDelegate.setCompatVectorFromResourcesEnabled(true);
+    }
```

Comment:
Thanks, I will check it. I will try either to avoid it or to move to depencency. Need to think more...